### PR TITLE
feat(chart): include CCI indicator

### PR DESCRIPTION
**Pull request by RobertKwiatkowski**
What have you changed and why?
CCI indicator file is added

Related Issues
https://github.com/rodrigo-brito/ninjabot/issues/110

### CODE REVIEW

File: plot/indicator/cci.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	return fmt.Sprintf("CCI(%d)", c.Period)
+}
+
+func (c cci) Overlay() bool {
+	return false
+}
+
+func (c *cci) Load(dataframe *model.Dataframe) {
+	c.Values = talib.Cci(dataframe.High, dataframe.Low, dataframe.Close, c.Period)
+	c.Time = dataframe.Time
```
Comment (rodrigo-brito) [inline plot/indicator/cci.go:37]: Can you remove the warmup perido?
```suggestion
	c.Time = dataframe.Time[c.Period:]
	c.Values = c.Values[c.Period:]
```

**Review by rodrigo-brito — CHANGES_REQUESTED** [review]
Thanks for the contribution

File: plot/indicator/cci.go (hunk 1 above)
Comment (RobertKwiatkowski) [reply plot/indicator/cci.go]: Done, sorry overlooked it

Comment (RobertKwiatkowski) [reply plot/indicator/cci.go]: Done, sorry overlooked it

File: plot/indicator/cci.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	"fmt"
+	"time"
+
+	"github.com/rodrigo-brito/ninjabot/model"
+	"github.com/rodrigo-brito/ninjabot/plot"
+
+	"github.com/markcheno/go-talib"
+)
+
+func Cci(period int, color string) plot.Indicator {
```
Comment (rodrigo-brito) [inline plot/indicator/cci.go:13]: The last change.
```suggestion
func CCI(period int, color string) plot.Indicator {
```

**Review by rodrigo-brito — CHANGES_REQUESTED** [review]

File: plot/indicator/cci.go (hunk 2 above)
Comment (RobertKwiatkowski) [reply plot/indicator/cci.go]: Done

**Review by rodrigo-brito — APPROVED** [review]

**Pull request merged**